diagnostic.md

```diff
@@ -32,23 +36,32 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+attr() - get or set values on HTML attributes.
+css() - get or set CSS attributes.
+addClass() and removeClass() - for adding and removing classes.
+val() - to get or set form field values.
 <!-- your answer ends here -->
```

Comment:
a little more specific please! in order for `css()` to work as a getter, there has to be some stuff inside the `()`, specifically `<some property>`. As a setter, `css()` requires `'<some property>', '<some value>'`.